public/javascripts/status_script.js

```diff
@@ -174,69 +173,72 @@ function UpdateDispatcher() {
 }
 
 function UpdateBootstrax() {
-    $.getJSON("status/get_bootstrax_status", function(data) {
-      if (data.length == 0)
+  var timeout = 20*1000; // seconds since last update
+  for (var i in eventbuilders) {
+    $.getJSON("status/get_bootstrax_status?host="+eventbuilders[i], (data) => {
+      var h = "#"+eventbuilders[i]+"_";
+      if (typeof data.host == 'undefined') {
+        $(h+"status").html("offline");
+        $(h+"status").css("color", "gray");
+        $(h+"target").html("");
```

Comment:
I recently change this to `targets`. Also, can add the `post_processing` argument?